module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <array>
+#include <utility>
+#include <vector>
+
+using Pixel = uint8_t;
+using WidthType = uint8_t;
+using HeightType = uint8_t;
```

Comment:
What's point of declaring 3 aliases for uint8_t?

Reply:
Because they mean something else :)